room/email_utils.py

```diff
@@ -2,25 +2,33 @@
 from django.core.mail import EmailMultiAlternatives
 from django.template.loader import render_to_string
 
+
 def send_invitation(to_email, code, room):
     from_email = settings.DEFAULT_EMAIL_FROM
     accept_url = settings.INVITE_ACCEPT_URL
-    
-    subject = 'Invitation to use TeamGroove'
-    text_content = f'Invitation to use TeamGroove service. Your code is: {code}' 
-    html_content = render_to_string('email_invitation.html', {'invitation_code': code, 'room': room, 'accept_url': accept_url})
+
+    subject = "Invitation to use TeamGroove"
+    text_content = f"Invitation to use TeamGroove service. Your code is: {code}"
+    html_content = render_to_string(
+        "email_invitation.html",
+        {"invitation_code": code, "room": room, "accept_url": accept_url},
+    )
 
     msg = EmailMultiAlternatives(subject, text_content, from_email, [to_email])
-    msg.attach_alternative(html_content, 'text/html')
+    msg.attach_alternative(html_content, "text/html")
     msg.send()
 
 
 def send_invitation_accepted(room, invitation):
     from_email = settings.DEFAULT_EMAIL_FROM
-    subject = 'Invitation accepted'
-    text_content = 'Your invitation was accepted'
-    html_content = render_to_string('email_accepted_invitation.html', {'room': room, 'invitation': invitation})
+    subject = "Invitation accepted"
+    text_content = "Your invitation was accepted"
+    html_content = render_to_string(
+        "email_accepted_invitation.html", {"room": room, "invitation": invitation}
```

Comment:
OK, we have no guidelines on this, so I make this as a suggestion. When breaking a function call across multiple lines, I recommend putting each argument onto it's own line like this:
```python
html_content = render_to_string(
    'email_accepted_invitation.html', 
    {'room': room, 'invitation': invitation}
)
```
It's easier to pick out the arguments and also easier to correctly indent them when they grow.

Reply:
I agree that your version is easier to read. 
This is a result of using black as our formatter. What was originally:

`html_content = render_to_string('email_accepted_invitation.html', {'room': room, 'invitation': invitation})`

Reply:
Black needs a rewrite.